string frame_id  # TF frame that 
Segment[] segments
